[cli] add command to set/get external FEM's LNA gain (#5563)
openthread/openthread: src/cli/README.md

> factoryreset
```

> ### fem
> 
> Get the external FEM parameters.
> 
> ```bash
> > fem
> LNA gain 11 dBm
> Done
> ```

Reviewer: ```suggestion
### fem lnagain

Get LNA gain in dBm of the external FEM.

```bash
> fem lnagain
11
Done
```
```
```suggestion
### fem

Get the external FEM parameters.

```bash
> fem
LNA gain 11 dBm
Done
```
```

Author: For this, I would like to use `fem` command to retrieve all the fem parameters so no subcommand is needed. How do you think?

Reviewer: Is it more consistent to have getter and setter the same pattern? I treat the CLI another protocol to control OpenThread. As you mentioned, the unit is already documented in API and README of the CLI. `LNA gain 11 dBm` uses more bytes and is harder to parse than `11` for scripts.

### fem lnagain \<LNA gain\>

> Set the external FEM parameters.

Reviewer: ```suggestion
Set the LNA gain in dBm of the external FEM.
```

> factoryreset
```

> ### femparam

Reviewer: ```suggestion
### fem lnagain
```

Get the external FEM parameters.

```bash

> > femparam
> LNA gain 11 dBm
> Done

Reviewer: ```suggestion
> fem lnagain
11
Done
```

Reviewer: I know there are examples in CLI that input is a number without unit, and the output is a number with unit. 
I prefer input and output to be the same for consistency. Thoughts? @jwhui @abtink @brianljt

Author: This is inline with ccathreshold and txpower. I think the unit related to power is dBm for most of the cases so there are no need to add unit to the input number. Also, the unit is clarified in the README.md file for cli.

```

> ### femparam \<LNA gain\>

Reviewer: ```suggestion
### fem lnagain \<LNA gain\>
```

Done
```

### fem

Get external FEM parameters.

```bash
> fem lnagain
LNA gain 11 dBm
Done
```

> 

Reviewer 2: Is there a use case for this? Are we planning to have other FEM parameters here?

If we want to keep this, is this covered by the expect tests?

Author: Per #5561, there are needs to set FEM gain to radio so that when setting/getting CCA threshold, user does not need to offset the LNA gain anymore. There might be other FEM parameters added later, like PA gain to cap the tx power.

This cli is covered in expect tests in cli-misc.exp

Reviewer 2: I see `fem lnagain` is covered, but not `fem`.

Note that the doc above seems incorrect for the example output. It should be `fem` with zero args, correct?

Author: Sorry, my mistake. Updated.
